src/components/NavigationBar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import './NavigationBar.css';
+
+import wbooksLogo from '../assets/wbooks_logo.svg';
+import notificationsSvg from '../assets/notifications.svg';
+import addBookSvg from '../assets/add_book.svg';
+
+export const NavigationBar = () => (
```

Comment:
you can use `export default NavigationBar = () => (` so in other files you can do:
```
import NavigationBar from './NavigationBar';
```
and avoid the named import